cli/generate.go

```diff
@@ -168,19 +189,45 @@ func generateMake() cli.Command {
 				return errors.Wrapf(err, "generating evergreen config from build file(s)")
 			}
 
-			output, err := json.MarshalIndent(conf, "", "\t")
-			if err != nil {
-				return errors.Wrap(err, "marshalling evergreen config as JSON")
-			}
-			if outputFile != "" {
-				if err := ioutil.WriteFile(outputFile, output, 0644); err != nil {
-					return errors.Wrapf(err, "writing JSON config to file '%s'", outputFile)
-				}
-			} else {
-				fmt.Println(string(output))
+			if err := formatAndOutputGeneratedConfig(conf, c.String(outputFormatFlagName), outputFile); err != nil {
```

Comment:
I think you are missing the `outputFormatFlagName` from the flags in this cli func

Reply:
Fixed.